### PR TITLE
PAYARA-3265 Openapi using @Operation(hidden = true)

### PR DESCRIPTION
**Pull request by Ykus**
I don't know, if this is the best solution. So I am open to any suggestions or comments.

### CODE REVIEW

File: appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/OpenApiService.java [hunk 1]
```diff
@@ -288,6 +289,7 @@ private OpenAPI buildDocument() throws OpenAPIBuildException {
                 openapi = new ApplicationProcessor(classes).process(openapi, appConfig);
                 openapi = new BaseProcessor(baseURLs).process(openapi, appConfig);
                 openapi = new FilterProcessor().process(openapi, appConfig);
+                openapi = new PathsProcessor().process(openapi, appConfig);
```
Comment (MattGill98) [inline appserver/payara-appserver-modules/microprofile/openapi/src/main/java/fish/payara/microprofile/openapi/impl/OpenApiService.java]: The extra processor here adds unneeded technical overhead in my opinion. I would personally add this functionality to the BaseProcessor class.

**Comment by Ykus** [comment]
Fixed. Thank you.

**Comment by MattGill98** [comment]
Thanks for those changes @Ykus. I've tested with the MicroProfile OpenAPI TCK and it still passes. Looks all good to me!

**Comment by MattGill98** [comment]
jenkins test please

**Comment by MattGill98** [comment]
jenkins test please

**Pull request merged**